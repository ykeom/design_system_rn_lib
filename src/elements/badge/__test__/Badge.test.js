import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Badge from '../Badge';

describe('Badge', () => {
  it('should render', () => {
    const component = shallow(<Badge />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
