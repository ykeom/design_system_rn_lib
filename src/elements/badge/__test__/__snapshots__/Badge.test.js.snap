// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    badge
  </Text>
</View>
`;
